### PR TITLE
feat(address) - address subfield labels are configurable

### PR DESCRIPTION
**Pull request by amrutha-rajiv**
## **Description**

- address subfield labels will b e configurable by providing address a `config` input
- config will be of the following format - 
```
config  = {
   address1: {
        label: 'Address'
   },
   address2: {
        label: 'Address line 2'
   },
   country:{
        label: 'Country'
   },
   zip: {
        label: 'Zip Code'
   },
   state: {
        label: 'State'
   },
}
```
- All keys are optional, defaults are retrieved from the labels service
- Config for each subfield will be enhanced in the future to allow properties like "required" or "hidden" for the ability to require or hide sub-fields.
#### **Verify that...**

- [ ] Any related demos where added and `npm start` still works
- [ ] New demos work in `Safari`, `Chrome` and `Firefox`
- [ ] `npm run lint` passes
- [ ] `npm test` passes and code coverage is increased
- [ ] `npm run compile` still works

##### **Screenshots**

### CODE REVIEW

**Comment by jgodi** [comment]
|File (4)|Change (:white_check_mark: +0.03%)|
|---|---|
|Control.ts|:white_check_mark: +0.00%|
|Address.ts|:white_check_mark: +4.94%|
|novo-label-service.ts|:white_check_mark: +0.53%|
|FormUtils.ts|:bangbang: -0.13%|

*Please note, individual file changes won't add up to the overall coverage due to number of lines!*

**All new files must be at 80% coverage and old files must NOT decrease in coverage!**

File: src/platform/elements/form/extras/address/Address.ts [hunk 1]
```diff
@@ -16,18 +18,19 @@ const ADDRESS_VALUE_ACCESSOR = {
...
+        <input type="text" class="apt suite" id="address2" name="address2" [placeholder]="config.address2.label" autocomplete="shipping address-line-2" [(ngModel)]="model.address2" (ngModelChange)="updateControl()"/>
+        <input type="text" class="city locality" id="city" name="city" [placeholder]="config.city.label" autocomplete="shipping city locality" [(ngModel)]="model.city" (ngModelChange)="updateControl()"/>
+        <novo-select class="state region" id="state" [options]="states" [placeholder]="config.state.label" autocomplete="shipping region" [(ngModel)]="model.state" (ngModelChange)="onStateChange($event)"></novo-select>
+        <input type="text" class="zip postal-code" id="zip" name="zip" [placeholder]="config.zip.label" autocomplete="shipping postal-code" [(ngModel)]="model.zip" (ngModelChange)="updateControl()"/>
+        <novo-select class="country-name" id="country" [options]="countries" [placeholder]="config.country.label" autocomplete="shipping country" [(ngModel)]="model.countryName" (ngModelChange)="onCountryChange($event)"></novo-select>
     `
 })
 export class NovoAddressElement implements ControlValueAccessor, OnInit {
+    @Input() config: any;
```
Comment (jgodi) [inline src/platform/elements/form/extras/address/Address.ts]: interface for this config?

Comment (amrutha-rajiv) [reply src/platform/elements/form/extras/address/Address.ts]: coming right up

**Comment by jgodi** [comment]
|File (4)|Change (:white_check_mark: +0.04%)|
|---|---|
|Control.ts|:white_check_mark: +0.00%|
|Address.ts|:white_check_mark: +4.94%|
|novo-label-service.ts|:white_check_mark: +0.53%|
|FormUtils.ts|:bangbang: -0.13%|

*Please note, individual file changes won't add up to the overall coverage due to number of lines!*

**All new files must be at 80% coverage and old files must NOT decrease in coverage!**

**Pull request merged**